src/components/textbox/textbox.pw.tsx

```diff
@@ -0,0 +1,1244 @@
+      mount,
+      page,
+    }) => {
+      await mount(
+        <TextboxComponent
+          labelInline
+          labelAlign="right"
+          validationOnLabel
+          {...{ [type]: "Message" }}
+        />
+      );
+
+      const label = getDataElementByValue(page, "label");
+
+      const labelParent = await label.evaluateHandle((element: Element) => {
+        return element.parentElement;
+      });
+
+      const dataElementExistsInLabelParent = await labelParent.evaluate(
+        (parentElement, selector) => {
+          const element = parentElement.querySelector(selector);
+          return element ? element.getAttribute("data-element") : null;
+        },
+        ICON
+      );
+
+      expect(dataElementExistsInLabelParent).toBe(type);
+    });
+  });
+
+  ([
+    [VALIDATION.ERROR, "error", true],
+    [VALIDATION.WARNING, "warning", true],
+    [VALIDATION.INFO, "info", true],
+    ["rgb(102, 132, 148)", "error", false],
+    ["rgb(102, 132, 148)", "warning", false],
+    ["rgb(102, 132, 148)", "info", false],
+  ] as [string, string, boolean][]).forEach(([borderColor, type, bool]) => {
+    test(`should render with ${borderColor} input border when validation type ${type} is ${bool}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <TextboxComponent
+          labelInline
+          labelAlign="right"
+          {...{ [type]: bool }}
+        />
+      );
+
+      await expect(textbox(page)).toHaveCSS("border-bottom-color", borderColor);
+    });
+  });
+
+  test("should render with leftChildren prop", async ({ mount, page }) => {
+    await mount(<TextboxComponentWithLeftChildren />);
+
+    await expect(textbox(page).locator(BUTTON)).toHaveAttribute(
+      "data-component",
+      "button"
+    );
+    await expect(textbox(page).locator(BUTTON)).toBeVisible();
+  });
+
+  testData.forEach((prefix) => {
+    test(`should render with prefix prop as ${prefix}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextboxComponent prefix={prefix} />);
+
+      await expect(textboxPrefix(page)).toHaveText(prefix);
+      await expect(textboxPrefix(page)).toHaveCSS("font-size", "14px");
+      await expect(textboxPrefix(page)).toHaveCSS("font-weight", "900");
+      await expect(textboxPrefix(page)).toHaveCSS("margin-left", "12px");
+    });
+  });
+
+  test("should render with prefix prop whilst align prop is also set to 'right'", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TextboxComponent prefix="foo" align="right" />);
+
+    await expect(textbox(page)).toHaveCSS("flex-direction", "row");
+  });
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
+  we can assert it exists in the DOM in the specified DOM structure, but cannot properly test its visibility
+  without added complexity or adding new data tags to `Textbox`s DOM. */
+
+  ([
+    [true, "input", "label"],
```

Comment:
**comment**: In terms of the tests themselves - I've not looked at it in detail but again I think this can be done a bit better using a single `evaluate` to get the DOM element using the various DOM queries, and then check that's truthy (since it seems we can't check visibility directly unless we have a playwright locator).

As you imply in the comment though, I'm not sure these tests add much value so I'd not be sad to see them removed 👍

Reply:
Yeah, going to just remove this one, test is adding no real value and its scenario is probably adequately covered already via unit test